rivershared/cmd/update-mod-go/main.go

```diff
@@ -0,0 +1,124 @@
+// Package main provides a command to help bump the `go/`toolchain` directives
```

Comment:
The recommended convention on `main` packages is to name this first word after the program name, rather than `main`: https://tip.golang.org/doc/comment#cmd

Reply:
Ah interesting, TIL. Updated both programs.